src/types/BulkUpload.ts

```diff
@@ -14,6 +14,7 @@ interface BulkUpload {
 	readonly id: number;
 	fileName: string;
 	sourceKey: string;
+	readonly createdBy: number;
```

Comment:
This should probably be moved to be next to `createdAt`

(also maybe we should put these in alphabetical order in general but whatever)